changelogs/8.0.asciidoc

```diff
@@ -3,12 +3,78 @@
+- Removed legacy Jaeger gRPC/HTTP endpoints {pull}6417[6417]
+- Removed `apm-server.jaeger` configuration options {pull}6560[6560]
+- Removed `apm-server.instrumentation` configuration options in favor of `instrumentation` {pull}6560[6560]
+- Removed `apm-server.rum.{allowed_service,event_rate}` configuration option in favor of `apm-server.auth.anonymous.{allow_service,rate_limit}` {pull}6560[6560]
+- Removed `apm-server.{api_key,secret_token}` configuration options in favor of `apm-server.auth.{api_key,secret_token}` {pull}6560[6560]
+- Removed unused `span.start.us` field, and deprecated `span.http.*` fields {pull}6602[6602]
+- Removed `apm-server.register.ingest.pipeline` and `output.elasticsearch.pipeline` configuration options {pull}6575[6575]
+- Removed `logging.ecs` and `logging.json` config {pull}6613[6613]
+- Removed `apm-server.data_streams.enabled`, and `setup.*` configuration options {pull}6606[6606]
+- Removed `apm-server.sampling.keep_unsampled` configuration option; non-RUM unsampled transactions are always dropped {pull}6514[6514] {pull}6669[6669]
+- Removed interpretation of non-standard "component" attribute from OpenTelemetry spans; changed default span type and subtypes {pull}6834[6834]
+- Onboarding documents are no longer indexed {pull}6431[6431]
+- APM Server now responds with 403 (HTTP) and PermissionDenied (gRPC) for authenticated but unauthorized requests {pull}5545[5545]
+- `sourcemap.error` and `sourcemap.updated` are no longer set due to failing to find a matching source map {pull}5631[5631]
+- Removed `service.name` from dataset {pull}5451[5451]
+
+[float]
+==== Added
+- Introduced a delete phase for all data streams. Traces, errors and logs are kept for 10 days, metrics are kept for 90 days {pull}6480[6480]
+- Changed RUM traces to use a dedicated data stream (`traces-apm.rum`). RUM traces are kept for 90 days {pull}6480[6480]
+- Standalone apm-server can now fetch source maps uploaded to Kibana, when `apm-server.kibana` is configured {pull}6447[6447]
+- Added metrics for new Elasticsearch output: `libbeat.output.events.{acked,batches,toomany}`; added tracing and log correlation {pull}6630[6630]
+- The `labels` indexed field is now ECS compliant (string only) and added a new `numeric_labels` object that holds labels with numeric values {pull}6633[6633]
+- APM Server is now using a new Elasticsearch output implementation {pull}6656[6656]
+- Modify default standalone apm-server config values to be more in line with the default managed apm-server values {pull}6675[6675]
+- APM Server now has beta support to receive OpenTelemetry Logs on the OTLP/GRPC receiver {pull}6768[6768]
+- experimental:[] Add support for CloudWatch metric streams through firehose endpoint {pull}6380[6380]
+- Upgrade Go to 1.16.5 {pull}5454[5454]
+- Add HTTP span fields as top level ECS fields {pull}5396[5396]
+- Introduce `apm-server.auth.anonymous.*` config {pull}5623[5623]
+- Upgrade Go to 1.16.6 {pull}5754[5754]
```

Comment:
Should we remove this old Upgrade to Go `1.16.x` entries and add the version that `8.0` will use? The current entries may be confusing for users going through the changelog.

Reply:
Good idea. Removed the other Go Upgrade entry as well.

```suggestion
- Add HTTP span fields as top level ECS fields {pull}5396[5396]
- Introduce `apm-server.auth.anonymous.*` config {pull}5623[5623]
- Upgrade Go to 1.17.6 {pull}7207[7207]
```